src/views/AppStore.vue

```diff
@@ -83,7 +83,7 @@
             app for Umbrel.
           </p>
           <b-link
-            href="https://github.com/getumbrel/umbrel/blob/master/apps/README.md"
+            href="https://github.com/runcitadel/compose-nonfree/tree/main/apps
```

Comment:
```suggestion
            href="https://github.com/runcitadel/compose-nonfree/blob/main/apps/README.md
```

Please keep the README link here.